src/java.base/share/classes/jdk/internal/foreign/StringSupport.java

```diff
@@ -33,6 +33,7 @@
 import jdk.internal.vm.annotation.ForceInline;
```

Comment:
Nitpick: update copyright year 😄

Reply:
Had it on my list but forgot it. Should be fixed now.